Formula/opensc.rb

```diff
@@ -33,6 +33,18 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<~EOS
+    The OpenSSH PKCS11 smartcard integration will not work from High Sierra
+    onwards. If you need this functionality, unlink this formula, then install
+    the OpenSC cask:
+
+    brew unlink opensc
+    brew cask install opensc
+
+    (See https://github.com/Homebrew/homebrew-core/issues/44786)
```

Comment:
```suggestion
```

I don't think anyone needs to be referred back to this issue. What do you think @SMillerDev?